docs/CHANGELOG.md

```diff
@@ -13,10 +13,14 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ### Changed
 
+* Bumped version of `vergen` to `3.0`. ([#1242])
+
```

Comment:
This is an internal change without any impact, not sure if it's worth listing here.

Reply:
true that, I did want to revisit that changelog discussion of what should be in/out, communicating breakages better etc.

Reply:
Took it out from the changelog